Update README.md

nixonrs-bucs opened the pull request:
As I'm updating the ReadMe, include any relative information useful for sprint 9

nixonrs-bucs committed:
Update README.md

As I'm updated the ReadMe, include any relative information useful for sprint 9

nixonrs-bucs committed:
Update README.md

Any 404 signs you see means I couldn't find the original author of the document.

ChristopherOaks reviewed (changes requested):
Could you please include any relevant information about the directory structure, how to start up the project, or anything in the documentation that might help with setting it up or understanding how things are organized. Clear instructions or notes on those aspects would be appreciated. And just to be clear, this project is definitely still intended for private development use only at the moment and is absolutely not for production use.

ChristopherOaks commented:
Gonna be honest, I feel like my paragraph response was just too harsh. my bad❤️❤️❤️❤️

nixonrs-bucs committed:
Update README.md

Still updating ReadMe, almost got the instructions done

nixonrs-bucs committed:
Update README.md

ChristopherOaks reviewed (dismissed):
I fw this heavy

nixonrs-bucs committed:
Update README.md

I deleted setting up a VPC, I don't know on keeping the security group because we had a tasked addressing the security group.

nixonrs-bucs committed:
Update README.md

nixonrs-bucs committed:
Update README.md

Took away the authors

nixonrs-bucs committed:
Update README.md

Made some minor edits

nixonrs-bucs committed:
Update README.md

Chicken Jockey for inspiration

nixonrs-bucs committed:
Update README.md

nixonrs-bucs committed:
Update README.md

nixonrs-bucs committed:
Update README.md

Still working on when deploying an instances on AWS. Needing to know the docker commands and git commands

BrofessorTec reviewed (approved):
As we update our documentation next sprint there may need to be some more changes to this (like how the "How to run locally" doc is from another group), but for now the structure looks pretty good and better than what we had

ChristopherOaks reviewed (approved):
i fw this heavy 2: electric boogaloo: the squeakal
